src/main/java/mate/academy/model/Ticket.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private MovieSession movieSession;
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

lets use FetchType.LAZY here as well